docs/chromadb-schema.md

```
@@ -0,0 +1,172 @@
+
+```bash
+cd backend
+npx tsx src/scripts/inspect-chroma.ts
+```
+
+**機能概要:**
+
+1.  **接続確認**: ローカルまたは Docker コンテナ上の ChromaDB に接続を試みます。
+2.  **コレクション統計**: 存在するコレクション名と、保存されているデータ総数 (`Items`) を表示します。
+3.  **データサンプリング**: データが存在する場合、設定された件数（`SAMPLE_LIMIT`、デフォルトはコード内で定義）のレコードを取得し、以下を表示します。
+  - ドキュメント内容 (`documents`)
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)